march_hardware/include/march_hardware/IMotionCube.h

```diff
@@ -81,8 +95,12 @@ class IMotionCube : public Slave
   void mapMosiPDOs();
   void writeInitialSettings(uint8_t cycle_time);
 
-  AbsoluteEncoder absolute_encoder_;
-  IncrementalEncoder incremental_encoder_;
+  // Use of smart pointers are necessary here to make dependency injection
+  // possible and thus allow for mocking the encoders. A unique pointer is
+  // is chosen since the IMotionCube should be the owner and the encoders
```

Comment:
```suggestion
  // possible and thus allow for mocking the encoders. A unique pointer is
  // chosen since the IMotionCube should be the owner and the encoders
```